src/inmanta/protocol/methods.py

```diff
@@ -259,7 +277,7 @@ def clear_environment(id: uuid.UUID):
     """
     Clear all data from this environment.
 
-    :param id: The uuid of the environment.
+    :param id: The id of the environment to be cleared.
```

Comment:
We need to be more precise in the description of this endpoint. Not all data is removed by this endpoint. The settings are kept.